docs/intro.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Introduction'
+weight: 20
+---
+
+[Glee][https://github.com/asyncapi/glee) is a spec-first framework that helps you build server-side applications. It leverages the AsyncAPI specification to make you more productive:
```

Comment:
```suggestion
[Glee](https://github.com/asyncapi/glee) is a spec-first framework that helps you build server-side applications. It leverages the AsyncAPI specification to make you more productive:
```
consider adding another paragraph explaining what actually `spec-first` means.

Reply:
Glee operates on the principle of defining the API specification (AsyncAPI) before diving into the actual implementation of the application logic. WDYT?